pages/services/percona-mongo/0.2.0-3.4.13/troubleshooting/index.md

```diff
@@ -29,7 +29,7 @@ You can also access the logs via the Mesos UI:
 
 <a name="accessing-metrics"></a>
 # Accessing Metrics
-_PROVIDE INSTRUCTIONS FOR ACCESSING METRICS OR INDICATE A LACK OF METRICS W/I LIMITATIONS.MD._
+Service-level metrics will be added in a future release of this service.
```

Comment:
@timvaillancourt What does this mean?   What is the level of effort involved?

Reply:
@ToddGreenstein : We believe a medium-high amount of effort is involved and we have an alternative/workaround via the Percona PMM monitoring functionality supported in this release. I will update this page to clarify.

I believe the plan is to add DC/OS metrics in our next release.